source/jobs/ra-2019-mtv.rst

```diff
@@ -0,0 +1,54 @@
+Research Assistant Opportunity (2019): Nuclear Facility Modeling for Nuclear Non-proliferation
+==============================================================================================
+
+*[Updated: March 27, 2019]*
+
+The Computational Nuclear Engineering Research Group (CNERG) is a participant
+in the Signals and Source Terms research thrust area of the `Consortium for
+Monitoring, Technology and Verification <http://mtv.engin.umich.edu>`_.  This
+thrust area will provide insight into the set of measurable quantities at
+various stages in the processing and handling of nuclear materials.
+
+CNERG researchers develop computational models of nuclear facilities with
+sufficient fidelity to simulate the time-varying nature of such measurable
+quantities as those facilities engage in normal and off-normal activities.
+
+CNERG is seeking a qualified individual to develop and validate such
+computational models.  Working in collaboration with other consortium
+partners, the primary measurables of interest will be identified and the
+models will be improved to increase the reliability of simulating those
+measurable quantities.
+xs
+Experience and Desired Skills
+------------------------------
+
+Candidates will be evaluated aginst the following criteria:
+
+* Undergraduate experience in Nuclear Engineering, Physics, or relevant other field
+* Experience with modeling and simulation of nuclear fuel cycle facilities
```

Comment:
same question: do you actually expect an incoming grad student to have this experience when they arrive? or can this be learned?